books/free-programming-books-bn.md

```diff
@@ -194,6 +195,9 @@
 
 * [TypeScript: All You Need To Know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVCzmP8m2l_h8PnVTSvcnJWO) - Stack Learner
 
+### Golang
```

Comment:
Wrong file line. It goes at L97, above `HTML`

Not a book. Related issue: #6591

Reply:
Resolved at Resolved at https://github.com/davorpa/free-programming-books/commit/145277376ad8ad32d82668a2c7f1ebd202dfe263